data/tutorials/language/1ds_01_arrays.md

```diff
@@ -0,0 +1,158 @@
+---
+id: arrays
+title: Arrays
+description: >
+  The standard library's Array module
+category: "Data Structures"
+---
+
+# Arrays
+
+## Introduction
+
+In OCaml, arrays are collections of elements of the same type that are stored in contiguous memory locations. They are mutable, each value can be updated, but they have a fixed size; which set them apart from lists.
```

Comment:
```suggestion
In OCaml, arrays are collections of elements of the same type that are stored in contiguous memory locations. They are mutable, each value can be updated, but since they have a fixed size, it sets them apart from lists.
```